tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -55,7 +55,7 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
         [InlineData(NetCoreApp21WebApp, "2.1")]
         [InlineData(NetCoreApp31MvcApp, "3.1")]
         [InlineData(NetCoreApp50MvcApp, "5.0")]
-        [InlineData(NetCore7PreviewMvcApp, "7.0.0-preview.7.22375.6")]
+        [InlineData(NetCore7PreviewMvcApp, "7.0.100-rc.1.22431.12")]
```

Comment:
stretch image, needs to use the constant we are creating. Same with the github action test right above

Reply:
Makes sense! For my own clarity -- these are the images that would have dotnet 7 prebaked and would not use dynamic installation. Correct?

Reply:
@snehapar9 The CLI images are the opposite, they only use dynamic installation.

I'm seeing `dotnet-7.0.100-preview.7.22377.5.tar.gz` in the [dev storage account](https://oryxsdksdev.blob.core.windows.net/dotnet?restype=container&comp=list&include=metadata)